web/src/installers/versions.js

```diff
@@ -87,6 +87,7 @@ module.exports.InstallerVersions = {
     "1.7.4",
   ],
   kotsadm: [
+    // auto-kotsadm-update
```

Comment:
not `"__KOTSADM_BINARY_VERSION__",`?

Reply:
Following convention here that it's the name of the job, e.g. :https://github.com/replicatedhq/kURL/blob/6abb99ac12abc5e07423d0e7a0c7811282a74260/web/src/installers/versions.js#L72-L73